subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApiSpecification.groovy

```diff
@@ -242,4 +242,12 @@ abstract class ToolingApiSpecification extends Specification {
     protected static GradleVersion getTargetVersion() {
         GradleVersion.version(targetDist.version.baseVersion.version)
     }
+
+    /**
+     * @return the current <b>base</b> version of Gradle being used to run this test
+     */
+    protected static GradleVersion getCurrentVersion() {
+        GradleVersion.current().baseVersion
+    }
+
```

Comment:
I'd make the version comparisons a method of this class. Maybe a String overload to make it more convenient. For example:

```
static boolean currentVersionSmallerThan(GradleVersion version) { ... }
static boolean currentVersionGreaterThanEquals(GradleVersion version) { ... }
```

Alternatively, I'd pull these version-specific methods in a new class instead of the parent class so it can be used anywhere else.